src/components/DpLabel/DpLabel.vue

```diff
@@ -14,25 +14,26 @@
           v-cleanhtml="h" />
       </span>
     </span>
-    <i
+    <dp-contextual-help
       v-if="tooltip !== ''"
-      :class="prefixClass('fa fa-question-circle u-mt-0_125 ml-auto')"
+      :class="prefixClass('mt-px ml-0.5')"
       :aria-label="ariaLabel"
-      v-tooltip="tooltip" />
+      :text="tooltip" />
   </label>
 </template>
 
 <script>
-import { CleanHtml, Tooltip } from '~/directives'
+import { CleanHtml } from '~/directives'
 import { de } from '~/components/shared/translations'
 import { prefixClassMixin } from '~/mixins'
+import DpContextualHelp from '~/components/DpContextualHelp/DpContextualHelp.vue'
```

Comment:
Does it also work to include this with

```suggestion
import DpContextualHelp from '~/components/DpContextualHelp'
```
?

Reply:
https://github.com/demos-europe/demosplan-ui/pull/851/commits/71ef394cfeb9ff8470151ba6a8c3a1d29b4ed80b